src/tooltip/README.md

```diff
@@ -0,0 +1,49 @@
+# @dojo/widgets/tooltip/Tooltip widget
+
+Dojo 2's `Tooltip` component can be used to show or hide content around a child component based on specific events.
+
+## Features
+
+- Agnostic to an eventing implementation
+- Supports top, right, bottom, and left orientations
+- Optionally persistent
+
+![Image of basic dialog](http://placekitten.com/450/300)
+
+## Themeing
+
+The following CSS classes are used to style the `Tooltip` widget and should be provided by custom themes:
+
+- `bottom`: Applied to the top-level wrapper node for a bottom-oriented tooltip
+- `content`: Applied to the node containing property-supplied tooltip content
+- `left`: Applied to the top-level wrapper node for a left-oriented tooltip
+- `right`: Applied to the top-level wrapper node for a right-oriented tooltip
+- `root`: Applied to the top-level wrapper node
+- `top`: Applied to the top-level wrapper node for a top-oriented tooltip
+
+## Example Usage
+
+*Basic Example*
+```typescript
+import TextInput from '@dojo/widgets/textinput/TextInput';
+import Tooltip from '@dojo/widgets/tooltip/Tooltip';
+import { w } from '@dojo/widget-core/d';
+
+w(Tooltip, {
+	key: 'foo',
+	content: 'This is a tooltip!',
+	showing: this._tooltipShowing
```

Comment:
do we use `showing` or `show` / `open` / `visible` elsewhere? not a fan of the `showing` naming here

Reply:
looks like `Dialog` and `SlidePane` use `open` ?

Reply:
A tooltip doesn't open or close, but I'll go with `open` for consistency.